packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -204,13 +204,22 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             return -1;
         }
 
+        if (maxBatchSize !== undefined) {
+            const bufferSize = this.messageBuffer.reduce<number>(
+                (acc, entry) => acc + (entry.contents.length as number), 0,
+            );
+            if (bufferSize + (message.contents.length as number) > maxBatchSize) {
+                this.flush();
```

Comment:
I think this would lead to interleaved batches though.. The batch would now be basically split on size and then the server would broadcast these batches, clients may see them interleaved and fail the batch consistency asserts inside the `ScheduleManager` as the clientId of the messages within a batch may change.

Reply:
It breaks 2. By default, all ops are batched implicitly, due to the FlushMode setting. There are batch semantics in ScheduleManager which validate that we are not processing interleaving batches. If we send them to the server outside of the batch semantics (see the batch property in the message metadata), we risk interleaving them.

Reply:
Reverting the first iteration of this pr in favor of changing a configuration value